docs/tutorials/ssl.md

```diff
@@ -12,31 +12,29 @@ mongoose.connect('mongodb://127.0.0.1:27017/test?ssl=true');
 The `ssl` option defaults to `false` for connection strings that start with `mongodb://`. However,
 the `ssl` option defaults to `true` for connection strings that start with `mongodb+srv://`. So if you are using an srv connection string to connect to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas), SSL is enabled by default.
```

Comment:
Is this still true? Does the `ssl` option default to `true` for `mongodb+srv://`, or does `tls` default to true?

Reply:
I am not sure about this. I suspect it does not default to true based on +srv://

Reply:
@vkarpov15 according to https://www.mongodb.com/docs/manual/reference/connection-string/ it still defaults to true based on mongodb+srv://

![image](https://github.com/Automattic/mongoose/assets/7505965/1774f2b8-c0e0-499a-ae0f-888ffe9f0590)

I have made another commit to make this clearer for the `tls` property